content/en/v3/admin/platforms/k3s/_index.md

```diff
@@ -160,6 +185,7 @@ This job will create the secrets in vault which will be used by external secrets
 
 - To verify the job succeeded, run `jx admin log`
 - To verfiy the secrets were created, run `kubectl get es -A` and `jx secret verify`
+- If this didn't work try and repeat the steps but commit your dummy changes through github repository directly without 
```

Comment:
Seems like this statement is incomplete, also the boot job runs 5 times on failure, so my guess is that in ur case, those were still old jobs, but nevertheless I think it's ok to add this.